README.md.jinja

```diff
@@ -5,6 +5,12 @@
 [![pipeline status]({{ gitlab_url }}/badges/{{ odoo_version }}/pipeline.svg)]({{ gitlab_url }}/commits/{{ odoo_version }})
 [![coverage report]({{ gitlab_url }}/badges/{{ odoo_version }}/coverage.svg)]({{ gitlab_url }}/commits/{{ odoo_version }})
 {%- endif %}
+{%- if domains_prod %}
+[![Deployment data](https://img.shields.io/badge/Deployment-prod-green)](https://{{ domains_prod[0].hosts[0] }})
```

Comment:
I think you better use the same as here: https://github.com/Tecnativa/doodba-copier-template/blob/788851afcd758e837f266867eea51a8cd46f40b2/prod.yaml.jinja#L33

OTOH, it could be `http` in some circumstances: https://github.com/Tecnativa/doodba-copier-template/blob/788851afcd758e837f266867eea51a8cd46f40b2/copier.yml#L231-L233

Reply:
Thanks! I hadn't seen that macro, much better now!

As we can't know if it will be `https` or not, I am leaving the link as `http`, as it will be redirected to secure if active.